packages/react/src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -76,17 +78,18 @@ const defaultProps = {
 const DEFAULT_PROGRESS_BAR_COLOR = blue60;
 
 const ProgressBar = ({
-  label,
+  className,
   helperText,
   hideLabel,
+  i18n,
+  label,
   light,
-  value,
-  valueUnit,
   max,
   renderIcon,
   renderIconByName,
   thresholds,
-  i18n,
+  value,
+  valueUnit,
```

Comment:
I just re-ordered these by alphabetical automatically. sorry

Reply:
Have I found a kindred spirit? I have thought about adding a prettier/eslint rule.

Reply:
I also like the alphabetical order, if we could have that automated it would be great. Same goes for the proptypes